docker/Makefile

```diff
@@ -3,41 +3,45 @@ all: latest v3.4.x v3.3.x v3.2.x
 latest:
 	docker build \
 		-t systemsgenetics/kinc:latest-cpu \
-		--build-arg NVIDIA_HEADLESS=1 \
-		--build-arg ACE_REVISION=develop \
+		--build-arg NVIDIA_HEADLESS=0 \
```

Comment:
The NVIDIA_HEADLESS settings are backwards now. It should be enabled for CPU build and disabled for GPU build. nvidia-headless is basically a stub for the actual nvidia drivers. Because of the way KINC works, you have to have the NVIDIA drivers installed on your system or else KINC will not run. So for CPU-only machines (which would use the CPU docker image), you would install the nvidia-headless drivers to keep KINC happy without having to install the actual NVIDIA drivers. This trick is also how you would run KINC natively on a CPU-only machine.

Reply:
Oh, why is called 'headless'?   Shouldn't it 'gpuless'?  But I'll switch it back.

Reply:
fixed in latest push to the branch